### ISSUE TITLE
The JSDOC that is created is not correctly indented

### ISSUE THREAD

**Issue by agamrafaeli** [mohseenrm/lit-it#1]
For instance:

```
class myClass {
    // Run lit-it here
    static myFunc(myParam, myParam2) {
        //Code...
    }
}
```

Results in this:

```
class myClass {
/**
* @function {function name}
* @param  {type} myParam  {description}
* @param  {type} myParam2 {description}
* @return {type} {description}
*/
    static myFunc(myParam, myParam2) {
        //Code...
    }
}
```

**Comment by mohseenrm** [comment]
Thanks for the description, will update you once it's fixed.

**Comment by mohseenrm** [comment]
Does not fix the general code block indentation issue

**Comment by mohseenrm** [comment]
PR #13, this issue has not been resolved, done some local testing, I'll try and patch this in, with the same PR

**Comment by mohseenrm** [comment]
~~@kylelamse can you checkout my branch and see why the test breaks?~~
Fixed with PR #13
